core/corehttp/logs.go

```diff
@@ -50,7 +50,7 @@ func LogOption() ServeOption {
 			w.WriteHeader(200)
 			wnf, errs := newWriteErrNotifier(w)
 			lwriter.WriterGroup.AddWriter(wnf)
-			log.Event(n.Context(), "log API client connected") //nolint deprecated
+			log.Info("log API client connected")
```

Comment:
Apparently we still need this...

> grep "log API client connected" log_out

fails.

I _thought_ we were sending events here now...